groups/infrastructure/profiles/live-eu-west-2/live/vars

```diff
@@ -0,0 +1,8 @@
+environment = "live"
+aws_profile = "live-eu-west-2"
+enable_container_insights = true
+
+ec2_instance_type = "m7i.2xlarge"
+asg_desired_instance_count = 2
```

Comment:
You're defining the `asg_desired_instance_count` without defining the `asg_max_instance_count`. This could lead to an invalid config.

When defining the max instance count, we need to take in to account the ECS rolling updates and build in capacity for that which will allow the ASG to temporarily scale up.